src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -189,6 +189,26 @@ private void CheckTrainingParameters(ImageClassificationEstimator.Options option
             return (jpegData, resizedImage);
         }
 
+
+        private static Tensor Encode(VBuffer<byte> buffer, int length)
```

Comment:
>length [](start = 63, length = 6)

It seems that this is always called with `buffer.Length`, so this argument is not needed.

Reply:
Thanks for pointing it out. I will make the change.